pandas/core/generic.py

```diff
@@ -7847,6 +7847,8 @@ def _where(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         --------
         :func:`DataFrame.%(name_other)s` : Return an object of same shape as
```

Comment:
Can you change `self` to `caller` both here and the line below?

Reply:
Absolutely. I also modified the notes, so that "caller" is used instead of "DataFrame" (made little sense in the context of Series). I hope that makes sense.